src/vcSettings.cpp

```diff
@@ -822,3 +830,22 @@ udResult vcSettings_UpdateLanguageOptions(vcSettings *pSetting)
 epilogue:
   return result;
 }
+
+void vcSettings_UpdateServerFlag(vcSettings *pSettings)
+{
+    size_t sLength = udStrlen(pSettings->maptiles.tileServerAddress);
+    pSettings->maptiles.serverFlag = vcMTSF_Count;
+    for (size_t i = 0; i < vcMTSF_Count; ++i)
+    {
+        if (udStrstr(pSettings->maptiles.tileServerAddress, sLength, g_vcMapTileServerName[i]) != 0)
```

Comment:
See [this comment](https://github.com/Euclideon/vaultclient/pull/209#discussion_r355850084) for why this needs to change.